spec/features/topical_event_spec.rb

```diff
@@ -30,6 +30,19 @@
     expect(page).to have_css("img[src='https://www.gov.uk/some-image.png'][alt='Text describing the image']")
   end
 
+  it "includes image srcset for various image sizes" do
+    visit base_path
+    expect(page).to have_css("img[srcset='https://www.gov.uk/some-medium-image.png 2x, https://www.gov.uk/some-high-image.png 3x']")
+  end
+
+  context "when the image has no variants" do
+    let(:content_item) { fetch_fixture("topical_event_with_no_image_variants") }
```

Comment:
I didn't realise collections had this `fetch_fixture` method. We usually something like [this](https://github.com/alphagov/collections/blob/a30feec0625d190724a5e765c400161ff646bfa2/spec/features/atom_feeds_spec.rb#L117)

Reply:
It isn't a blocker for this PR, but maybe you could update your spec at some point!

Reply:
oh interesting, the `GovukSchemas::Example.find(schema_name, example_name: schema_name)` seems to fetch test fixtures from the examples written in publishing_api. It basically makes these tests become sort of contract tests between publishing api schema and collections app. however that then means that the test data has to be written in the examples folder of publishing api and therefore makes management of these testes a bit strange.

I can prepare a separate pr for that as that would require a change in publishing api to provide specific exampel files there.

Reply:
You can either create a new example file in publishing components (which might not be a bad idea, as there's only one example for topical events which is quite basic [ie](https://github.com/alphagov/publishing-api/tree/main/content_schemas/examples/topical_event/frontend))

Or you can just use the existing schema, and mutate the JSON response like in that link I posted. In that way you don't need to make a change in publishing api.

Reply:
Step 7 on here! https://github.com/alphagov/publishing-api/blob/main/docs/content_schemas/changing-an-existing-content-schema.md#adding-a-new-field-to-a-content-schema

Reply:
hmm, i'm not exited to set up test data in another repo for this repo to use. If the example files were desired to be used as contract tests - then publishing_api CI workflow should include an action that executes any tests in Collections app that depend on those example files. Otherwise a dev who changes the schema or examples, would not be aware of collections application breaking.

I did however do as you suggested, i removed the extra fixture file i had added and just modified the data in place.

Reply:
This was a good catch @hannako. I have to admit I didn't delve into what `fetch_fixture` was doing too closely. Using the examples content items is a pattern that is reflected in other applications, so I agree that it would make sense to use it here too. 

As a bit of historical context, the content used to exist in a separate gem, but I believe they were merged into publishing-api to remove our app overhead.